### ISSUE TITLE
Output from python code in HTML is not getting rendered, it is only getting rendered as js in the browser console. I'm using PyScript link.

### ISSUE THREAD

**Issue by AbhinavJoe** [pyscript/pyscript#1929]
# Description
I came accross PyScript about 24 hours ago. Since then I've been trying to make it work and render my python code output on the main HTML page but it just doesn't render instead renders in the browser's console window. There was a similar issue I found Issue #427. In that they said to clear browser cache and all, I did and it was still not working.
In the end, I decided to raise an issue. I hope my issue can get clarified.
P.S. I tried running PyScript using the links in 2 other systems as well and it was showing the same error in console and not working properly.

## Code Snippet
- Initially I tried it with my main HTML and Python file and it was not working. The code for that is shown below:
![Screenshot (117)](https://github.com/pyscript/pyscript/assets/91539403/b9a46886-1f97-47f9-b116-b38c4987c243)

- So, I created sample files to see if any basic implementation were working. They were not.
Below is the basic implementation I used:
![Screenshot (115)](https://github.com/pyscript/pyscript/assets/91539403/e42f19fa-cc4d-489a-8c7d-32b351d53016)
Again, the output was showing only in the console of the browser and not the main page.

## Output Screenshot
- Below is the screenshot of the output for the first code snippet provided above:
![Screenshot (109)](https://github.com/pyscript/pyscript/assets/91539403/5f592c4a-69ad-4508-8b94-53e4bb1c46e2)
- There is no output in the main page but rather in the console.
- The console showed two errors as shown below.

### Error 1:
![Screenshot (113)](https://github.com/pyscript/pyscript/assets/91539403/9a2231de-bb74-45f3-a212-29432f7d541f)

### Error 2:
![Screenshot (110)](https://github.com/pyscript/pyscript/assets/91539403/a064e9bc-39d2-4d9b-98be-cb6d16479a69)

**Comment by JeffersGlass** [comment]
`print` goes to the JS console by design. To show content on the page, you want to use `pyscript.display`:

```py
from pyscript import display
display("hello world!")
```

This is bevhaing as intended, but is clearly giving you a bad experience - we have a chance to improve our user experience here, for sure.

**Comment by AbhinavJoe** [comment]
Thanks for your help @JeffersGlass 
I don't know how but I skipped the display module when I was going through the documentation. Plus, there was a youtube video  (an year old video) that showed python output directly on the browser without any specific pyscript modules so, I guess I got confused by that as well.
I will have to change some code for my main project to make it work but it is doable thanks!
